test/test_cardgames.py

```diff
@@ -0,0 +1,23 @@
+from testing_base import *
+
+
+def printAllPlayerCards_test(self):
+  deck1 = Deck()
+  player1 = Player("john")
+  player2 = Player("jeff")
+  player3 = Player("george")
+  dealer1 = Dealer()
+
+
+  dealer1.dealCards(2, [player1], deck1)
+  dealer1.dealCards(3, [player2], deck1)
+  dealer1.dealCards(4, [player3], deck1)
```

Comment:
If you want, you could condense this into one line by making a list of players (if you are okay with giving each player an equal amount of cards